sass/directives/_headers.scss

```diff
@@ -60,39 +60,14 @@
   }
 }
 
-@mixin header-hero-product($background-image-large, $background-image-medium: nil) {
+@mixin header-hero {
   position: relative;
-  min-height: $header-height-product;
   overflow: hidden;
   background-color: $base-background-color;
-  background-image: url(asset_path(#{$background-image-large}));
-  background-size: auto $header-height-product;
-  background-position: right bottom;
-  background-repeat: repeat no-repeat;
-
-  @media only screen and (max-width: $medium-screen-max) {
-    background-image: url(asset_path(#{$background-image-medium}));
-  }
 
   @media only screen and (max-width: $small-screen-max) {
     @include display(flex);
     @include align-items(center);
-    min-height: unset;
```

Comment:
Try cleaning up the header stylesheet in employer first, then check to see if you've kept any headers with min-height. If there's nothing left that already has it and needs to keep it, you can move it and then test to make sure the headers all still look right